src/care/host_ptr.h

```diff
@@ -17,6 +17,11 @@
 // Std library headers
 #include <cstddef>
 
+#if defined(CARE_ENABLE_STALE_DATA_CHECK) && !defined(CHAI_DISABLE_RM) && (defined(CHAI_ENABLE_CUDA) || defined(CHAI_ENABLE_HIP) || defined(CHAI_ENABLE_GPU_SIMULATION_MODE))
```

Comment:
This is optional, but you could make a variable defined as defined(CARE_ENABLE_STALE_DATA_CHECK) && !defined(CHAI_DISABLE_RM) && (defined(CHAI_ENABLE_CUDA) || defined(CHAI_ENABLE_HIP) || defined(CHAI_ENABLE_GPU_SIMULATION_MODE)) since it is long and you use it twice

Reply:
I'm not going to worry about this since it is already in one application's fork of CARE.